bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -213,6 +214,13 @@ private constructor(
     @get:JvmName("oauth")
     val oauth: OAuthMethods by lazy { OAuthMethods(this) }
 
+    /**
+     * Access API methods under "oembed" endpoint.
```

Comment:
Just a really minor comment on this KDoc here: We should make these comments consistent over all endpoints listed in here. Sometimes we write **Access API methods under "api/vX/notification[s]" endpoint.** and sometimes it is only **Access API methods under "oembed" endpoint.** I am happy with both as long as we are consistent.

Reply:
I’ve unified them all. I’ve also sorted them alphabetically again. To make reviewing that easier, I have created separate commits for both.